evap/staff/forms.py

```diff
@@ -1063,6 +1064,21 @@ class Meta:
         }
 
 
+class InfotextForm(forms.ModelForm):
+    class Meta:
+        model = Infotext
+        fields = ("title_de", "title_en", "content_de", "content_en")
+
+    def validate_unique(self):
+        super().validate_unique()
+
+        # add empty error to all empty fields to show the user that they are required
```

Comment:
I don't get it, there are some errors, but we need to add more? What kind of error is it that makes `self.errors` true?

Reply:
Its the error of the failed CheckConstraint, which creates an error on the whole model, not individual empty fields

Reply:
Out of boredom, I added it in commit `nothalfempty constraint`